### DISCUSSION TITLE
More extensive keyboard shortcuts

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9477] [Ideas]
> Created by: Seamus (seamus@gridworkdesign.com) on 2015/10/19 23:54:15 +0000
> Votes at time of UserVoice import: 2

-----------------------------------

I'd like to have keyboard shortcuts to all the most common parts of the control panel. In ExpressionEngine I did this by injecting a little bit of javascript that added accesskey attributes to the buttons and links I used a lot (fields, sections, categories, create new, save, members, etc). It's invaluable being able to bounce around setting things up without ever needing to mouse around.

**Comment by seamusmh** [comment]
```
$(".buttons.right .add").attr("accesskey", "n");
$(".btngroup.submit .add").attr("accesskey", "n");
$(".submit.add").attr("accesskey", "n");
$(".livepreviewbtn").attr("accesskey", "l");
$("body").append( "<a href='/admin/settings/fields' accesskey='f'></a>" );
$("body").append( "<a href='/admin/settings/sections' accesskey='c'></a>" );
$("body").append( "<a href='/admin/settings/categories' accesskey='g'></a>" );
$("body").append( "<a href='/admin/settings/plugins' accesskey='m'></a>" );
$("body").append( "<a href='/admin/entries' accesskey='e'></a>" );
$("body").append( "<a href='/admin/assets' accesskey='a'></a>" );
$("body").append( "<a href='/admin/settings' accesskey='s'></a>" );
$("body").append( "<a href='/admin/utilities' accesskey='u'></a>" );
$("body").append( "<a href='/admin/feed-me' accesskey='i'></a>" );

```

**Comment by brandonkelly** [comment]
We were just talking about this yesterday.

**Comment by seamusmh** [comment]
Sympatico. The accessibility/UX improvements in 3.5 are so great, hard not to start imagining all the glorious ways you can keep going…
